hewr/R/to_hub_quantile_table.R

```diff
@@ -0,0 +1,141 @@
+    "other_ed_visits" = "inc other ed visits",
+    "prop_disease_ed_visits" =
+      glue::glue("inc {disease_abbr} prop ed visits")
+  )
+
+
+  get_location_table <- function(model_run_dir) {
+    samples_paths <- fs::dir_ls(model_run_dir,
+      recurse = TRUE,
+      glob = "*/samples.parquet"
+    )
+    quantiles_paths <- fs::dir_ls(model_run_dir,
+      recurse = TRUE,
+      glob = "*_quantiles.parquet"
+    )
+
+    quantilized_samples_forecast <- samples_paths |>
+      purrr::map(arrow::read_parquet) |>
+      purrr::map(\(x) {
+        forecasttools::trajectories_to_quantiles(x,
+          timepoint_cols = "date",
+          value_col = ".value",
+          id_cols = setdiff(colnames(x), c(
+            "date", ".value", ".draw",
+            ".chain", ".iteration"
+          ))
+        )
+      }) |>
+      tibble::enframe(name = "file_path", value = "data") |>
+      dplyr::mutate(model = .data$file_path |>
+        fs::path_dir() |>
+        fs::path_file()) |>
+      dplyr::mutate(model = dplyr::if_else(.data$model == "timeseries_e",
+        "baseline_ts", .data$model
+      )) |>
+      dplyr::select("model", "data")
+
+    quantiles_forecast <- quantiles_paths |>
+      purrr::map(\(x) {
+        arrow::read_parquet(x) |>
+          dplyr::rename("quantile_value" = ".value")
+      }) |>
+      tibble::enframe(name = "file_path", value = "data") |>
+      dplyr::mutate(model = "baseline_cdc") |>
+      dplyr::select("model", "data")
+
+    forecast_data <-
+      dplyr::bind_rows(
+        quantilized_samples_forecast,
+        quantiles_forecast
+      ) |>
+      tidyr::unnest("data") |>
+      dplyr::filter(.data$.variable %in% names(variable_target_key)) |>
+      dplyr::mutate(target_prefix = dplyr::if_else(
+        .data$resolution == "epiweekly", "wk ", ""
+      )) |>
+      dplyr::mutate(target_core = variable_target_key[.data$.variable]) |>
+      dplyr::mutate(target = stringr::str_c(
+        .data$target_prefix,
+        .data$target_core
+      )) |>
+      dplyr::mutate(reference_date = report_date) |>
+      dplyr::mutate(horizon_timescale = "days") |>
+      dplyr::mutate(horizon = forecasttools::horizons_from_target_end_dates(
+        reference_date = .data$reference_date,
+        horizon_timescale = .data$horizon_timescale,
+        target_end_dates = .data$date
+      )) |>
+      dplyr::mutate(
+        output_type = "quantile",
+        output_type_id = round(.data$quantile_level,
+          digits = 4
+        )
+      ) |>
+      dplyr::mutate(model_id = glue::glue(
+        "{model}_{resolution}",
+        "{dplyr::if_else(vctrs::vec_equal(",
+        "aggregated_numerator,TRUE, na_equal = TRUE),'_agg_num', '')}",
+        "{dplyr::if_else(vctrs::vec_equal(",
+        "aggregated_denominator, TRUE, na_equal = TRUE), '_agg_denom', '')}"
+      )) |>
+      dplyr::select(
+        "model_id",
+        "model",
+        "output_type",
+        "output_type_id",
+        "value" = "quantile_value",
```

Comment:
This is also umambiguous and what we recommend here: https://potential-adventure-637em36.pages.github.io/r_coding_tips/#use-bare-column-names-when-they-are-on-the-left-hand-side-of-an

```suggestion
        value = "quantile_value",
```

I do get that mixing strings colnames and and bare kwargs isn't the prettiest